automation/README.md

```diff
@@ -0,0 +1,67 @@
+# Automation with oVirt CI
+
+In order to keep playbooks in this repository operational,
+it is being integrated with [oVirt CI System][ovirt-ci-system-doc].
+Everything located under `automation` directory is related to integration
+to [oVirt CI System][ovirt-ci-system-doc].
+
+There is list of events which can trigger job on [oVirt CI System][ovirt-ci-system-doc],
+The KubeVirt Ansible repository is using `check-patch` event at the moment,
+to verify incoming patches.
```

Comment:
<pre>
 At present, the KubeVirt Ansible repository <b>uses the</b> `check-patch` event to verify incoming patches.
</pre>

Reply:
Addressed